packages/workspace/src/browser/workspace-service.spec.ts

```diff
@@ -483,43 +484,6 @@ describe('WorkspaceService', () => {
             await wsService.addRoot(new URI(folderB.uri));
             expect(spyWriteFile.calledWith(workspaceFileStat, { folders: [{ path: folderA.uri }, { path: folderB.uri }] })).to.be.true;
         });
-
-        [true, false].forEach(existTemporaryWorkspaceFile => {
```

Comment:
@elaihau i've removed it. It's flaks since it tests too much of implementation. Partially replaced with `add untitled`.